acid/features/auth/service.py

```diff
@@ -20,6 +23,17 @@ def drop_user_session():
     session.pop('user')
 
 
+def login_required(wrapped_function):
```

Comment:
Maybe rename it to ``admin_required``? It checks not only login.

Reply:
Agree.